qa/common/gen_qa_model_repository

```diff
@@ -77,11 +80,13 @@ cp ./gen_qa_models.py $HOST_SRCDIR/.
 cp ./gen_qa_zero_models.py $HOST_SRCDIR/.
 cp ./gen_qa_reshape_models.py $HOST_SRCDIR/.
 cp ./gen_qa_sequence_models.py $HOST_SRCDIR/.
+cp ./gen_ensemble_model_utils.py $HOST_SRCDIR/.
```

Comment:
Why isn't it just called "gen_qa_ensemble_models.py"? Its confusing that the utils used by other scripts is also a main script itself. Can you break apart that functionality.